src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetTransferSenderTransactionDetails/GetTransferTransactionDetailsQueryHandler.cs

```diff
@@ -0,0 +1,66 @@
+﻿using MediatR;
+using SFA.DAS.EAS.Application.Hashing;
+using SFA.DAS.EAS.Domain.Extensions;
+using SFA.DAS.EAS.Domain.Models.Transfers;
+using SFA.DAS.EAS.Infrastructure.Data;
+using SFA.DAS.EAS.Infrastructure.Extensions;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EAS.Application.Queries.GetTransferSenderTransactionDetails
+{
+    public class GetTransferTransactionDetailsQueryHandler : IAsyncRequestHandler<GetTransferTransactionDetailsQuery, GetTransferTransactionDetailsResponse>
+    {
+        private readonly EmployerFinancialDbContext _dbContext;
+        private readonly IPublicHashingService _publicHashingService;
+
+        public GetTransferTransactionDetailsQueryHandler(EmployerFinancialDbContext dbContext,
+            IPublicHashingService publicHashingService)
+        {
+            _dbContext = dbContext;
+            _publicHashingService = publicHashingService;
+        }
+
+        public async Task<GetTransferTransactionDetailsResponse> Handle(GetTransferTransactionDetailsQuery query)
+        {
+            var result = await _dbContext.GetTransfersByTargetAccountId(
```

Comment:
You could use entity framework and automapper here and project to a DTO that looks something like:
```
public class TransferDto
{
    public string CourseName { get; set; }
    public int ApprenticeCount { get; set; }
    public decimal PaymentTotal { get; set; }
    public AccountDto SenderAccount { get; set; }
    public AccountDto ReceiverAccount { get; set; }
}
```